src/lib/local-gateway.ts

```diff
@@ -0,0 +1,33 @@
+import { uiLogger } from './logger.js'
+
+export const localGwUrl = 'http://127.0.0.1:8080'
```

Comment:
Since we know it is not subdomains fault, we may want to switch to `localhost` which is Secure Context:

```suggestion
export const localGwUrl = 'http://localhost:8080'
```

Should be fine, as long  `fetch` follows redirects by default, or is there more to it?

Reply:
Based on https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts 127.0.0.1 should also be Secure Context. 

Is there any particular benefit to using localhost. We don't change the `redirect` config in fetch, so it's follow, and indeed it is followed.  

I don't have a strong opinion, just curious why localhost is preferred.

Reply:
I don't think there's any more to it for most usecases.